create_graphics.py

```diff
@@ -722,3 +726,7 @@ def graphics_driver(cla):
         if name not in ['specs', 'sites']:
             print(f"{name:>15s}: {val}")
     graphics_driver(CLARGS)
+
+
+if __name__ == '__main__':
+    check_args(sys.argv[1:])
```

Comment:
Since this is the main function of create_graphics, it may be slightly easier to follow if the name of this function is `create_graphics`.

Reply:
That makes sense. Is this something we should change now or in a subsequent PR?

Reply:
Do you mind pushing it to this one?

Reply:
Just pushed a new commit. Github should automatically update the PR, correct?

Reply:
That's right. :)